MultiSource/Benchmarks/ASC_Sequoia/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
 add_subdirectory(AMGmk)
 add_subdirectory(CrystalMk)
-add_subdirectory(IRSmk)
+# The IRSmk test fails after the Clang 11 upgrade.
```

Comment:
TODO: if the current ADO DevTest LNT build passes, revert changes to CMake files. The Makefile changes are the ones that disable the failing tests.